CHANGES.txt

```diff
@@ -5,6 +5,10 @@ Changes for Crate
 Unreleased
 ==========
 
+ - Fix: Array comparison with subselect required two pair of parenthesis.
```

Comment:
Imho there is no need to declare this as a fix. It's an improvement and not really a bug.